boltdesignsystem/bolt
# DS-599: Optimized Campaign Landing Page

## Jira

https://pegadigitalit.atlassian.net/browse/DS-599

## Summary

Created the demo pages for the Optimized Campaign Landing Page

## Details

- Created doc pages for the Optimized Campaign Landing Page templates
- Create doc pages for the Optimized Campaign Landing Page themes
- Modified the Layout CSS to fix a mobile bug
- Text + Icon spacing adjusted in Button and Text Link elements (due to Icon element handling white space better than Icon component)

## How to test

- Pull down the branch and look at the Optimized Campaign Landing Page templates in the doc site
- Reveal the Patternlab test folder and look at the examples of the themes
- Check for regressions on the Layout Component (this PR modifies the Layout Item width behavior so we need some deep checks here).
- Review the Video Thumbnail Center Component (we can change the name if we need) docs

## Release notes

- Added the Optimized Campaign Landing Page docs pages
### Visual Changes
#### Button Element
- Slightly increased the gap between button text and icon
#### Text Link Element
- Slightly increased the gap between link text and icon
